packages/preview-middleware-client/test/__mock__/sap/m/Popover.ts

```diff
@@ -0,0 +1,19 @@
+import Control from 'sap/ui/core/Control';
+
+export const getContentMock = jest.fn();
+
+export default class Popover{
+
+    constructor() {
+
+    }
+
+    static getContent() {
+        return getContentMock()
+    }
+
+    openBy(_oControl: Control | HTMLElement) {
```

Comment:
Remove Hungarian notation. Also, since you are not using this oControl anywhere but want the type safety you can just put `_`.